### PR TITLE
user creation display_name validation

### PR DESCRIPTION
**Pull request by rottabonus**
## prevent user creation without display_name

Disable moving forward on form when display_name is not valid

displayNameValid

### CODE REVIEW

**Review by idolgov — APPROVED** [review]

**Review by veeraalt — DISMISSED** [review]
Good job!

File: src/Screens/Onboarding/DisplayName.tsx [hunk 1]
```diff
@@ -35,6 +35,9 @@ const DisplayName = ({ navigation }: Props) => {
     });
   };
 
+  const isValidDisplayName =
+    displayName.length >= 2 && displayName.length <= 100;
```
Comment (ahto) [inline src/Screens/Onboarding/DisplayName.tsx:39]: The validation logic could be in its own function and that could be unit tested, but these are very simple rules.

**Review by ahto — APPROVED** [review]

File: src/Screens/Onboarding/DisplayName.tsx (hunk 1 above)
Comment (rottabonus) [reply src/Screens/Onboarding/DisplayName.tsx:39]: Yes, it is better to do like that. I'll make a commit in the following days :p

**Review by ahto — APPROVED** [review]

**Comment by Osamah-learn** [comment]
Good Job Bro

**Pull request merged**